onsite-library.json

```diff
@@ -0,0 +1,14 @@
+{
+  "docker": {
+    "packageRules": [
+      {
+        "allowedVersions": "<=1.7",
+        "packageNames": ["traefik"]
+      }
+    ]
+  },
```

Comment:
This is already part of `defaults.json`, so it doesn't need to be set again here.

Reply:
No, I moved it here. 😉